docs/blog/2019-11-25-introducing-gatsby-preview-for-cosmic-js/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Introducing Gatsby Preview for Cosmic JS"
+date: 2019-11-25
+author: Tony Spiro
+excerpt: "We’re excited to announce the official release of Gatsby Preview for Cosmic JS that enables you to add powerful content preview functionality to your Cosmic JS Gatsby website."
+tags:
+  - Cosmic JS
+  - Headless CMS
+  - Gatsby Preview
+---
+We’re excited to announce the official release of Gatsby Preview for [Cosmic JS](https://www.cosmicjs.com). This new integration enables you to add powerful content preview functionality to your Cosmic JS Gatsby website.
+
+## Getting Started
+
+1. First go to the [Gatsby Getting Started Page](https://www.gatsbyjs.com/get-started/) and follow the links for Cosmic JS.
+2. Select the [Cosmic JS-powered Gatsby Blog](https://www.cosmicjs.com/apps/gatsby-blog)
+3. You will then be asked for your GitHub and Cosmic JS authentication, so make sure you have both accounts set up ([sign up for Cosmic JS here](https://app.cosmicjs.com/signup)).
+4. After onboarding, go to any Post in your Cosmic JS Bucket and find the big, beautiful, purple button to preview your content.
+![](./cosmic-js-gatsby-preview-screenshot.jpg)
+
+
+#### What happens?
+After clicking the “Open Preview” button, a new tab will open to instantly show you what this content looks like for this specific version on your Gatsby website. This simple one-click preview will greatly enhance your content team’s content-creation workflow.
+
+![](./cosmic-js-gatsby-preview-screenshot.gif)
+
+
+
+#### How did this integration happen?
+The Cosmic JS / Gatsby Preview integration is the result of close collaboration between both teams to make the integration as easy as possible. Both sides had to refactor a bit of existing functionality to make it seamless. And the feedback that came from the collaboration effort actually helped to improve both products. We’re happy to say this powerful feature can be integrated in just a few clicks.
+
+#### Why is this a big deal?
+Both Gatsby and Cosmic JS believe deeply in providing developers the best development experience possible. But it’s not just about developers. Non-technical team members are crucial to building great web products. And one of the most requested features for content creators is preview, but this has been a challenge for developer teams pushing for highly-optimized JAMStack websites built on Gatsby.
+
+Gatsby Preview solves this problem by providing a simple button for content creators to experience an instant preview of content directly in the CMS dashboard. Gatsby preview with the [Cosmic JS headless CMS](https://www.cosmicjs.com/headless-cms), is a further extension to empower non-technical content creators to preview content quickly and easily, giving teams faster content marketing velocity.
```

Comment:
See above comment about non-technical. "content creators" without the caveat would work well here.